src/Common/constants.tsx

```diff
@@ -375,16 +375,16 @@ export const RESPIRATORY_SUPPORT = [
   { id: "NONE", text: "UNKNOWN" },
 ];
 
-export type PatientCategoryID = "Comfort" | "Stable" | "Moderate" | "Critical";
+export type PatientCategoryID = "Comfort" | "Mild" | "Moderate" | "Critical";
 
 export const PATIENT_CATEGORIES: {
   id: PatientCategoryID;
   text: PatientCategory;
   twClass: string;
 }[] = [
   { id: "Comfort", text: "Comfort Care", twClass: "patient-comfort" },
-  { id: "Stable", text: "Stable", twClass: "patient-stable" },
-  { id: "Moderate", text: "Abnormal", twClass: "patient-abnormal" },
+  { id: "Mild", text: "Mild", twClass: "patient-mild" },
+  { id: "Moderate", text: "Moderate", twClass: "patient-moderate" },
```

Comment:
You need not change the ID' and tailwind classess. You can just change the labels (text)